### DISCUSSION TITLE
Do you have a cheat sheet on setting up VSC and Code4i

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#3037] [Q&A]
I had to replace my PC.
I installed VSC.
I installed the required code for i modules.
I followed the instructions in the Getting Started.
I configured my LPARs.
I am unable to connect.

I cannot find how to download the password files for each LPAR to use with VSC/SSH. How do I do this? Its been more than 18 months since I did this.

Is there a video on setting up VSC.

Is there a cheat sheet on the steps required to complete the setup?

Thanks
Darryl Freinkel.

<hr />

⚠️ **REMOVE THIS LINE AND ANY SENSITIVE INFORMATION BELOW!** ⚠️

|Context|Version|
|-|-|
|Code for IBM i version|2.13.0|
|Visual Studio Code version|1.93.1|
|Operating System|win32_x64|

<details>
<summary>Active extensions</summary>

```
COBOL (cobol): 24.9.18
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.5.0
Db2 for IBM i (vscode-db2i): 1.5.1
Error Lens (errorlens): 3.20.0
Git (git): 1.0.0
Git Base (git-base): 1.0.0
GitHub (github): 0.0.1
Merge Conflict (merge-conflict): 1.0.0
Node Debug Auto-attach (debug-auto-launch): 1.0.0
TODO Highlight (vscode-todo-highlight): 1.0.5
```
</details>

<hr />

**_Not connected_** 🔌

**Comment by SJLennon** [comment]
@a4ginatl 

No knowledge of a cheat sheet or a video, but the documentation is pretty straight forward.

Some thoughts....
1) Are you connecting with a password or a private key?
2) The SSH Daemon is started. (STRTCPSVR *SSHD starts the daemon.)
3) You have created a connection? (If not, see the [Login ](https://codefori.github.io/docs/login/)section of the documentation.)

If you are still can't connect, the maintainers will want the OUPUT channel "Code for IBM i" complete info right after you try to connect. 
![image](https://github.com/user-attachments/assets/6c82d287-f066-441a-8f3e-42fac4d34d7f)
The screen shot is from my connection to PUB400.COM. The info is quite long and they like to be able to search it. You can copy and paste it into this issue.

**Comment by a4ginatl** [comment]
I finally found a video put out by Liam some time ago. Code4i has changed quite a bit and that’s good.

 

I have a network issue that the network folk are resolving, but it may take a while.

 

What I could not find is how to download the private key so h=that I can install it. I need the instruction on how to install the public key.

 

Thanks

Darryl Freinkel

A4G

Telephone: 770.321.8562

**Comment by SJLennon** [comment]
@a4ginatl 
I created my private and public keys with [PuttyGen](https://www.ssh.com/academy/ssh/putty/windows/puttygen). The public key I pasted into /home/lennons/.ssh/authorized_keys.  The private key I saved on my PC calling it PUB400.ppk and put the complete path to that in the Private Key field in my Code for IBM i connection.
